python/interpret_community/widget/templates/inlineDashboard.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
```

Comment:
what is the new file above:  python/interpret_community/widget/static/v0.2.js
is that needed?

Reply:
I guess this is used by the CDN somehow?

Reply:
it was for uploading to the cdn, not meant to be permanent